src/docs/tutorial/index.md

```diff
@@ -0,0 +1,28 @@
+---
+followup:
+  subtext: How do you want to use Taskcluster?
+  links:
+    tutorial/what-is-tc: I'm not sure - what is it?
+    tutorial/hello-world: I want to run a task manually in Taskcluster.
+    tutorial/gecko-tasks: I'm a Firefox/Gecko developer and I want to change builds or tests.
+    tutorial/debug-task: A task is failing and I want to debug it.
+    tutorial/apis: I want to work with the Taskcluster APIs.
+    tutorial/hack-tc: I want to hack on Taskcluster itself.
+---
+
+# Welcome to Taskcluster!
+
+---
+
+This tutorial is designed to help you learn what you need to know about
+Taskcluster to accomplish your goals. You can navigate it by answering the
+question at the bottom of each page, which will bring you to the most helpful
+information.
+
+If you want a more comprehensive description of Taskcluster's design and
+operation, see the manual portion of this documentation.
+
+## Patches Welcome!
+
+As you learn more, please feel welcome to [contribute](https://github.com/taskcluster/taskcluster-docs) more choices and pathways to this tutorial.
```

Comment:
The `contribute` link probably should lead to this repository?

Reply:
Good catch :) Everything under `src/docs` was copied from `taskcluster/docs` as is using a script. We can fix those later.